hack/release/make-release-tag.sh

```diff
@@ -57,4 +61,8 @@ $(git shortlog "$OLDVERS..HEAD")
 EOF
 
 printf "Created tag '%s'\n" "$NEWVERS"
-printf "Run '%s' to push the tag if you are happy\n" "git push --tags"
+
+printf "Testing whether tag '%s' can be pushed\n" "$NEWVERS"
+git push --dry-run origin "$NEWVERS"
```

Comment:
For me `origin` is my personal fork. I know for you @jpeach it's the upstream projectcontour repo. Not sure how to solve that in this script.

Reply:
I added some code to fish out a likely-looking remote. Does that work correctly for your config?

Reply:
That looks like a reasonable guess to me.